core/src/main/java/com/microsoft/applicationinsights/FormattedDuration.java

```diff
@@ -84,36 +84,36 @@ private static void appendDaysHoursMinutesSeconds(StringBuilder sb, long days, l
 
     private static void appendMinTwoDigits(StringBuilder sb, long value) {
         if (value < 10) {
-            sb.append("0");
+            sb.append('0');
```

Comment:
you can also append int directly.  wonder which is faster.

Reply:
this is to left pad with zero, e.g. `04` instead of `4`